build/package.json

```diff
@@ -88,6 +82,7 @@
   },
   "resolutions": {
     "json-schema": "0.4.0",
-    "jsonwebtoken": "9.0.0"
+    "jsonwebtoken": "9.0.0",
+    "minimist": "1.2.6"
```

Comment:
What's this for? It might be the cause of the issue that @lewis-sanchez was seeing, and in general we should avoid resolutions except in extreme circumstances.

Reply:
It was added to see if it helps resolve issue.. but didn't make a diff